definitions/variable/emissions/tag_geological-storage.yaml

```diff
@@ -0,0 +1,12 @@
+- Geogological Storage Source:
+    - Bioenergy:
+        description: geological storage of CO2 from biogenic orgin, i.e.,
+          bioenergy with carbon capture and storage (BECCS)
+    - Direct Air Capture:
+        description: direct air capture in combination with carbon-capture and
+          sequestration (CCS)
+    - Synthetic Fuels:
```

Comment:
It would be useful to provide more context here in terms of how this is different than bioenergy (bioenergy-based synthetic fuels are also present in scenarios)

Reply:
I switched from "Carbon-Neutral" to "Synthetic" to be less ambiguous... Plus there's a category "Bioenergy". Do you think clearly adding "synthetic fuels not including biofuels" would be sufficient? Or do you have a better suggestion for changing the variable name or extending the description?

Reply:
I would propose the following:

```
Bioenergy:
  description: geological storage of CO2 captured from bioenergy, i.e. energy carriers whose energy content is derived from biomass. This is also referred to as bioenergy with carbon capture and storage (BECCS). 
 [...]
 Synthetic Fuels:
   description: geological storage of CO2 captured from burning synthetic fuels produced from carbon-capture-and-utilization (CCU)
```

Reply:
I recently suggested "Carbon-neutral Efuels" to avoid confusion with biofuels (that are sometimes also categorized as synthetic fuels) as well as Efuels from fossil sources. Another option would be to only call them Efuels, and move the carbon-neutral to the description